docs/pages/attributesJSON.rst

```diff
@@ -29,7 +29,7 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
 
 "physics_simulator"
 	- string
-	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+	- What physics engine should be used for dynamics simulation. Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
 "gravity"
 	- 3-vector
 	- The default gravity to use for physical modeling. This can be overridden by Stage attributes.
```

Comment:
And where do we describe what a "Stage" is?

Reply:
We should upload a terminology page to the documentation.